CMakeLists.txt

```diff
@@ -4,15 +4,19 @@ project(torc
         VERSION 0.1
         LANGUAGES CXX)
 
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Is there a reason you upgraded to C++ 20? Below I set the standard to 17, and if we don't need 20, then I think we should stay at 17.

Reply:
Not really, I was just testing on this branch trying to get CppAD to compile in debug mode...